db/schema.rb

```diff
@@ -10,8 +10,45 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.1].define(version: 0) do
+ActiveRecord::Schema[7.1].define(version: 2023_11_29_103335) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "comments", force: :cascade do |t|
+    t.bigint "user_id", null: false
+    t.bigint "post_id", null: false
+    t.text "text"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["post_id"], name: "index_comments_on_post_id"
+    t.index ["user_id"], name: "index_comments_on_user_id"
+  end
+
+  create_table "likes", force: :cascade do |t|
+    t.bigint "user_id", null: false
+    t.bigint "post_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["post_id"], name: "index_likes_on_post_id"
+    t.index ["user_id"], name: "index_likes_on_user_id"
+  end
+
+  create_table "posts", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "users", force: :cascade do |t|
+    t.string "Name"
+    t.string "Photo"
+    t.text "Bio"
+    t.integer "PostCounter"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
```

Comment:
- [ ] **[OPTIONAL]** The migration file update works. But you would need to reflect this change for yourself. It works on my end. You need to run `rails db:migrate` to update the users' table.

Reply:
@julie-ify Thank you so much for the feedback. I ran rails db:migrate but there aren't any changes. Do you have any further recommendations? Morerover, the app works from my end.

Reply:
@julie-ify Thanks so much for your thorough review and feedback. i have manually updated the changes to the schema tables.